.github/workflows/deploy.yml

```diff
@@ -1,54 +1,79 @@
 name: Deploy to GitHub Pages
 
 on:
-  # Nightly @ 5AM UTC
-  schedule:
-    - cron: '0 5 * * *'
```

Comment:
I can't think of any reason why this needs to run on a schedule; I can re-add if I've overlooked something

Reply:
I believe this was added to run and deploy the [config script](https://github.com/openfga/openfga.dev/blob/main/scripts/update-config-page.mjs) periodically so that we don't need to worry about manually updating the config page when new OpenFGA releases are cut.